src/g/process.rs

```diff
@@ -29,72 +30,74 @@ pub async fn process_create_migration(model: &str) {
         println!("! {:?}", why.kind());
     });
     let file_path = String::from(file_dir) + &filename + ".rs";
-    let file_content = format!(
-        "use entity::{};
-use sea_orm::{{DbBackend, EntityTrait, Schema}};
-use sea_orm_migration::prelude::*;
 
-pub struct Migration;
+    // create tokens used in the quasi-templating below
+    let model_name_ident = format_ident!("{}", model);
+    let file_name_lit: syn::LitStr = syn::parse_str(&filename).expect("Unable to parse println!() argument");
```

Comment:
line 36 gets this error below;

`thread 'main' panicked at 'Unable to parse println!() argument: Error("expected string literal")'`

Can you try `cargo run g model test` command to test if this line works?
Thank you!

Reply:
sure I think I will find time over the weekend